src/mldb/s3_utils.py

```diff
@@ -79,7 +79,7 @@ def key_exists(bucket, key):
     # Return true if a key exists in s3 bucket
     # TODO: return None from the get functions if the key doesn't exist?
     #       (this would avoid one round-trip to S3)
-    client = get_s3_client()
+    client = get_s3_client_google()
```

Comment:
thanks for the contribution! could we adjust this so it first tries `get_s3_client()`, and if that fails, it falls back to `get_s3_client_google()`? that would allow the full testbed to be available once vasa comes back online.

Reply:
Resolved!